server/base/src/main/java/org/apache/accumulo/server/AccumuloDataVersion.java

```diff
@@ -81,6 +71,28 @@ public static int get() {
     return CURRENT_VERSION;
   }
 
-  public static final Set<Integer> CAN_RUN =
-      Set.of(SHORTEN_RFILE_KEYS, CRYPTO_CHANGES, ROOT_TABLET_META_CHANGES, CURRENT_VERSION);
+  public static final Set<Integer> CAN_RUN = Set.of(ROOT_TABLET_META_CHANGES, CURRENT_VERSION);
+
+  /**
+   * Get the stored, current working version.
+   *
+   * @param context the server context
+   * @return the stored data version
+   */
+  public static int getCurrentVersion(ServerContext context) {
+    int cv =
+        context.getServerDirs().getAccumuloPersistentVersion(context.getVolumeManager().getFirst());
+    ServerContext.ensureDataVersionCompatible(cv);
+    return cv;
+  }
+
+  public static String dataVersionToReleaseName(final int version) {
+    switch (version) {
+      case ROOT_TABLET_META_CHANGES:
+        return "2.1";
+      case REMOVE_DEPRECATIONS_FOR_VERSION_3:
+        return "3.0";
```

Comment:
It's highly unlikely, but possible that a data version could change in a bugfix release. Also, because data version can span multiple minor versions, it may not make sense to only show the minor release here, as that could imply it only applies to versions matching `a.b.*`, when it might also apply to version `a.b+1.*` as well. So, instead, let's use the full name of the version of the first appearance of the change.

```suggestion
        return "2.1.0";
      case REMOVE_DEPRECATIONS_FOR_VERSION_3:
        return "3.0.0";
```